server/controllers/api.py

```diff
@@ -923,6 +932,19 @@ def post(self, user, backup_id):
 
         return self.schema.store_comment(user, backup)
 
+    @marshal_with(schema.get_fields)
+    def get(self, user, backup_id):
+        backup = models.Backup.query.get(backup_id)
+        if not backup:
+            if user.is_admin:
+                restful.abort(404)
+            else:
+                restful.abort(403)
+        if not models.Backup.can(backup, user, "view"):
+            restful.abort(403)
+        return {"comments": [backup.comments]}
```

Comment:
A couple questions about testing:

- Once I've logged in, how do I log out? The nav bar seems to be broken, although I do see a link to logout written in the code that's appearing to be a logout link: 
<img width="1277" alt="screen shot 2017-03-08 at 1 26 57 am" src="https://cloud.githubusercontent.com/assets/13214848/23692733/734e4ece-039e-11e7-8994-eb80252e55a1.png">

- How do you go about setting up the testing database from the browser and how do you get a backup_id afterwards? I know that there are some backups seeded into the database during setup, but I'm not sure how to get a backup_id from the browser to put into the url that you've suggested. 

- How do you run the test files in the tests folder? using the command 
`python tests/test_api.py` didn't work for me last time.